packages/@vue/cli/bin/vue.js

```diff
@@ -65,6 +67,24 @@ program
   .option('-b, --bare', 'Scaffold project without beginner instructions')
   .action((name, cmd) => {
     const options = cleanArgs(cmd)
+
```

Comment:
Actuall we have [minimist](https://github.com/substack/minimist) in dependency, why not just use `minimist` to parse args ? Something like:

```suggestion
if (minimist(process.argv.slice(3))._.length > 1) {
  // do warning here
}
```